frontend/src/pages/Q&A/Q&A.jsx

```diff
@@ -132,16 +132,17 @@ function Ques(props) {
         setToastMessage("Q&A added successfully!");
         setOpenToast(true);
         setSeverity("success");
+        setFormData({
+          title: "",
+          description: "",
+          tags: [],
+        });
+        setCheckedState(new Array(Tags.length).fill(false));
+        DeactiveButton();
```

Comment:
what is this doing?

Reply:
1. this is function which closes the question pop up , so i called it when question is submitted successfully 
2. therefore after successful question submission it will close the question popup